// This file is part of KWIVER, and is distributed under the
// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.

syntax = "proto2";

package kwiver.protobuf;

message bounding_box {
  required double xmin = 1;
  required double ymin = 2;
  required double xmax = 3;
  required double ymax = 4;
}
